### DISCUSSION TITLE
GraphGym doesn't always log precision, recall and F1 score

### DISCUSSION THREAD

**Discussion by Rajjaa** [pyg-team/pytorch_geometric#4844] [Q&A] [answered]
Hello,

I am using GraphGym to run graph classification experiments.
I noticed that GraphGym logs the accuracy, precision, recall, and F1 score when using the MUTAG (from TUDataset) but it only logs the accuracy when using the CIFAR10 dataset (from GNNBenchmarkDataset). I couldn't find where the logged metrics are specified. Could you please show me how I can specify the metrics I want to log?

Thanks

**Comment by rusty1s** [comment, marked as answer]
This is handled [here](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/graphgym/logger.py#L110-L138), and cannot be customized currently. We will use accuracy, precision, recall, f1 and AUC for binary classification, and only use accuracy for multi-class classification.

**Reply by Rajjaa** [reply]
Thank you for the answer, this is helpful. I think can customize it by cutomizing the train and passing a custom LoggerCallback.
I'll keep you posted.
